.reek.yml

```diff
@@ -10,3 +10,4 @@ detectors:
 
 exclude_paths:
   - db/migrate
+  - app/controllers/gifts_controller.rb
```

Comment:
Do not exclude the controller from all controls. Just exclude it from the ones we don't want to run